src/command_modules/azure-cli-iotcentral/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.1.4
++++++
+* Use latest azure-mgmt-iotcentral version.
```

Comment:
You can leave this off, a cli user doesn't need to know this

Reply:
removed